### PR TITLE
rebalance pipe gun recipes

### PR DESCRIPTION
**Pull request by dissociativity**
#### Summary
SUMMARY: Balance "Rebalance higher end pipegun recipes in tool requirements and time taken, add lever action shotgun that is more or or less the heavy survivor carbine with an internal 6 round magazine that fires standard 12ga shotgun shells."

#### Purpose of change

Recipes seemed extremely quick for complex pipe guns and didn't require any welding at all, where oddly the .30-06 single shot pipe rifle did

#### Describe the solution

increase tool requirements and time taken for high end pipe weapons , add lever action shotgun as a higher end crafted shotgun option, still measurably worse than any standard pump action or semi auto shotgun, compareable to the heavy lever action .308 carbine in this regard.

#### Describe alternatives you've considered

leave as is, add lever action shotgun

#### Testing

load game, fix any json issues, make sure recipe shows up as intended, wish in lever action shotgun, and ammo, reload and fire away!

### CODE REVIEW

File: data/json/recipes/recipe_weapon.json [hunk 1]
```diff
@@ -852,6 +852,35 @@
...
+      { "id": "GLARE", "level": 2 }
+    ],
+    "tools": [
+      [
+        [ "oxy_torch", 30 ],
+        [ "welder", 150 ],
+        [ "large_repairkit", 75 ],
+        [ "small_repairkit", 150 ],
+        [ "welder_crude", 225 ],
+        [ "toolset", 225 ]
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_weapon.json:879]: This should be a requirement, if it isn't already

File: data/json/recipes/recipe_weapon.json [hunk 2]
```diff
@@ -852,6 +852,35 @@
...
+    "type": "recipe",
+    "result": "lever_shotgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ "gun", 3 ],
+    "difficulty": 7,
+    "time": "3 h",
+    "reversible": true,
+    "autolearn": true,
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_weapon.json:865]: This (and the other recipes) probably shouldn't be autolearned, but something you can pick up from a relevant book.

File: data/json/recipes/recipe_weapon.json (hunk 1 above)
Comment (dissociativity) [reply data/json/recipes/recipe_weapon.json:879]: I copied it from a recipe that's already around, I'll look it up and clean up that one too while I'm at it.

**Comment by ymber** [comment]
All of the rifle caliber pipe guns should go. Looking at the pressure rating chart for standard pipes (https://ipspipe.com/wp-content/uploads/2015/06/Pipe_pressure_rating.pdf) none of them are viable. This new lever action shotgun recipe is also far too simple. Look at a parts diagram for an M1887 for an idea of how complex the action is in these guns.

**Comment by dissociativity** [comment]
> 
> 
> All of the rifle caliber pipe guns should go. Looking at the pressure rating chart for standard pipes (https://ipspipe.com/wp-content/uploads/2015/06/Pipe_pressure_rating.pdf) none of them are viable. This new lever action shotgun recipe is also far too simple. Look at a parts diagram for an M1887 for an idea of how complex the action is in these guns.

preparing a PR for obsoletion of those, I'll ask for the okay from more people before pushing it as I don't feel up to the task of this level of data analysis at the moment

**Comment by dissociativity** [comment]
alright, I'l PR that and see how nightmarish and what issues it has in the morning, for now it's sleep time, can easily revert if there's too much disagreement over obsoletion after all.

Made sure it touched on generic guns too.

**Comment by dissociativity** [comment]
This PR is done as far as I'm concerned, I'll add more pipe guns that are realistic and practical to make in a seperate PR, a .357 repeater is naturally obligatory ; )

**Pull request merged**